model/driver/src/icon4py/model/driver/test_cases/gauss3d.py

```diff
@@ -90,13 +95,13 @@ def model_initialization_gauss3d(
     )
     end_cell_end = grid.end_index(cell_domain(h_grid.Zone.END))
 
-    w_numpy = xp.zeros((num_cells, num_levels + 1), dtype=float)
-    exner_numpy = xp.zeros((num_cells, num_levels), dtype=float)
-    rho_numpy = xp.zeros((num_cells, num_levels), dtype=float)
-    temperature_numpy = xp.zeros((num_cells, num_levels), dtype=float)
-    pressure_numpy = xp.zeros((num_cells, num_levels), dtype=float)
-    theta_v_numpy = xp.zeros((num_cells, num_levels), dtype=float)
-    eta_v_numpy = xp.zeros((num_cells, num_levels), dtype=float)
+    w_ndarray = xp.zeros((num_cells, num_levels + 1), dtype=float)
```

Comment:
```suggestion
    w_ndarray = np.zeros((num_cells, num_levels + 1), dtype=float)
```
?

Reply:
Thanks for the comments. 
I declare the `xp` via `xp = driver_config.host_or_device_array(backend)` and try to scrap the dependency on `xp` in `settings.py`. This is something I am not sure whether it is a good idea to do that or not. Perhaps I should have a pytest fixture xp that switches between cp and np depending whether the backend is on CPU or GPU.